File: script_testing/sample.py
#!/usr/local/bin/python

import sys
print 'I am sample Python Script'

if len(sys.argv ) < 2 :
    print 'Usage : at least one arg required!'
    sys.exit(2)

for arg in sys.argv:
    print arg

if sys.argv[1] != 0:
    print 'I would exit with non zero status!'
    sys.exit( int(sys.argv[1]) )
sys.exit(0)

File: script_testing/sample.pl
#!/usr/bin/perl -w 
print "I am sample perl script!\n" ;
$num_args = $#ARGV + 1;
if ($num_args < 1) {
    print "Usage: sample.pl arg1 arg2 arg3\n";
    exit 2;
}
foreach my $arg (@ARGV){
    print "$arg\n"
}

if ( $ARGV[0] != 0 ){
    print "I would exit with non zero status!\n" ;
    exit int($ARGV[0]);
}
exit 0;